.prettierignore

```diff
@@ -1,5 +1,5 @@
-src/generated/
-review/*.api.md
-*.yml
-*.yaml
-*.d.ts
+**/src/generated/
```

Comment:
Wait.. what!

Reply:
These changes seem unrelated to the PR. Maybe rebase the branch

Reply:
oh, thank you!